src/OnChain.ts

```diff
@@ -217,16 +323,17 @@ export const OnChainMixin = (superclass) =>
             throw new TransactionRestrictedError(error, { logger: onchainLogger })
           }
 
-          const { lnd } = getActiveOnchainLnd()
+          // const { lnd } = getActiveOnchainLnd()
```

Comment:
@dolcalmi Notice that this is the same call that is now in line 90 (this.client = getActiveOnchainLnd().lnd), so it should work just like it used to.